python/src/uagents/context.py

```diff
@@ -554,3 +558,11 @@ async def send_raw_exchange_envelope(
             destination=destination,
             endpoint="",
         )
+
+    async def send_wallet_message(
+        self,
+        destination: str,
+        text: str,
+        msg_type: int = 1,
+    ):
+        await self._wallet_messaging_client.send(destination, text, msg_type)
```

Comment:
We should check if the wallet client is None and/or add other checks as this will be the main interface for the developer.

Reply:
Good point, fixed.